lint381/c.py

```diff
@@ -11,6 +11,20 @@
 
 
 @linter.register
+@with_matched_tokens(start=match_regex("^sizeof$"), end=match_regex("^char$"))
+def prohibited_unsigned_char(source, *, match):
+    """Flag sizeof(char) as it defined to be 1 and is redundant."""
+    error_message = "sizeof(char) should not be used as it is defined by the "\
+                    "C standard to be equal to 1"
+
+    if match[1].value == "char":
+        yield Error(error_message, tokens=[match[0], match[1]])
+    elif (match[2].value == "char" and match[1].value == "("):
+        yield Error(error_message, tokens=[match[0], match[1],
```

Comment:
Consider using slicing, i.e. `match[0:3]` or `match[0:-1]`, if that's what you're trying to convey here.